doc/changelog/02-specification-language/11390-master+fix-9412-context-implicits.rst

```diff
@@ -0,0 +1,4 @@
+- **Fixed:**
+  Implicit arguments were ignored in :cmd:`Context`
```

Comment:
Please add a `.. warning::` box that this has been observed to be a breaking change in several projects and explaining the various strategies to restore compatibility.